framework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -0,0 +1,454 @@
+from utils import xmlUtils
+import Files
+import DataObjects
+from ..ValidationBase import ValidationBase
+#import Distributions
+#import MetricDistributor
+#from .ValidationBase import ValidationBase
+#from Models.PostProcessors import PostProcessor
+#Internal Modules End--------------------------------------------------------------------------------
+
+class PPDSS(ValidationBase):
+  """
+    DSS Scaling class.
+  """
+  @classmethod
+  def getInputSpecification(cls):
+    """
+      Method to get a reference to a class that specifies the input data for
+      class cls.
+      @ In, cls, the class for which we are retrieving the specification
+      @ Out, inputSpecification, InputData.ParameterInput, class to use for
+        specifying input of cls.
+    """
+    inputSpecification = super(PPDSS, cls).getInputSpecification()
+    #featuresInput = InputData.parameterInputFactory("Features", contentType=InputTypes.StringListType)
+    #featuresInput.addParam("type", InputTypes.StringType)
+    #inputSpecification.addSub(featuresInput)
+    #targetsInput = InputData.parameterInputFactory("Targets", contentType=InputTypes.StringListType)
+    #targetsInput.addParam("type", InputTypes.StringType)
+    #inputSpecification.addSub(targetsInput)
+    multiOutputInput = InputData.parameterInputFactory("multiOutput", contentType=InputTypes.StringType)
+    inputSpecification.addSub(multiOutputInput)
+    multiOutput = InputTypes.makeEnumType('MultiOutput', 'MultiOutputType', 'raw_values')
+    multiOutputInput = InputData.parameterInputFactory("multiOutput", contentType=multiOutput)
+    inputSpecification.addSub(multiOutputInput)
+    #pivotParameterInput = InputData.parameterInputFactory("pivotParameter", contentType=InputTypes.StringType)
+    #inputSpecification.addSub(pivotParameterInput)
+    #
+    # Have added the new pivotParameters for feature and target. The original has been commented out.
+    pivotParameterFeatureInput = InputData.parameterInputFactory("pivotParameterFeature", contentType=InputTypes.StringType)
+    inputSpecification.addSub(pivotParameterFeatureInput)
+    pivotParameterTargetInput = InputData.parameterInputFactory("pivotParameterTarget", contentType=InputTypes.StringType)
+    inputSpecification.addSub(pivotParameterTargetInput)
+    #metricInput = InputData.parameterInputFactory("Metric", contentType=InputTypes.StringType)
+    #metricInput.addParam("class", InputTypes.StringType, True)
+    #metricInput.addParam("type", InputTypes.StringType, True)
+    #inputSpecification.addSub(metricInput)
+    scaleTypeInput = InputData.parameterInputFactory("scale", contentType=InputTypes.StringType)
+    inputSpecification.addSub(scaleTypeInput)
+    scaleRatioBetaInput = InputData.parameterInputFactory("scaleBeta", contentType=InputTypes.FloatListType)
+    inputSpecification.addSub(scaleRatioBetaInput)
+    scaleRatioOmegaInput = InputData.parameterInputFactory("scaleOmega", contentType=InputTypes.FloatListType)
```

Comment:
please use "descr" key words to add the descriptions for each node, and attribute in the XML.